tests/test_make.py

```diff
@@ -1,61 +0,0 @@
-#!/usr/bin/env python
```

Comment:
So continuing on the CI tangent, has this file been move somewhere and/or is it being run?